docker/nodejs/docker-compose.yml

```diff
@@ -0,0 +1,20 @@
+version: "2"
+
+services:
+  api:
+    build: .
+    image: map_point_server
+    volumes:
+      - .:/usr/src/app/
+    ports:
+      - "3000:3000"
+    links:
+      - mongo
+    environment:
+      MONGO_URI: mongodb://mongo:27017/
+  mongo:
+    image: mongo
+    volumes:
+       - /data/db
```

Comment:
Should we have two options for this for older implementations of MongoDB (-v2.0) where `/data/db` is in `/var/lib/mongodb/`?

Reply:
Yeah I think we can add that in as a comment that way it can just be swapped.